browser-test/src/support/applicant_questions.ts

```diff
@@ -116,4 +116,8 @@ export class ApplicantQuestions {
     expect(await this.page.innerHTML('head'))
       .toContain('<meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">');
   }
+
+  async seeStaticQuestion(questionText: string){
+      expect(await this.page.textContent('html')).toContain(questionText);

```

Comment:
nit - maybe a more specific selector for textContent if there's a stable container?